### DISCUSSION TITLE
export large number of images

### DISCUSSION THREAD

**Discussion by julien134** [microsoft/PlanetaryComputer#151] [Q&A]
I currently use Google Earth Engine to apply a processing to a large number (50000) Sentinel-1 satellite images and then export them to a cloud bucket, in order to finally download them for local usage. Each processing/export task takes around 3 minutes, leading to a total processing duration of more than 3 months. 

Would MPC be faster than GEE for such export tasks ?

**Comment by TomAugspurger** [comment]
> Would MPC be faster than GEE for such export tasks ?

I'm not sure offhand, but I would encourage you to think about whether you really need to actually download the data. In our experience, it's much better to compute on the data directly where it's at (making sure your compute is in the same Azure region as the data: West Europe).
